docs/src/main/sphinx/release/release-369.md

```diff
@@ -0,0 +1,147 @@
+
+## ClickHouse connector
+
+* Upgrade minimum required version to 21.3. ({issue}`10703`)
+* Add support for [renaming schemas](/sql/alter-schema). ({issue}`10558`)
+* Add support for setting [column comments](/sql/comment). ({issue}`10641`)
+* Map ClickHouse `ipv4` and `ipv6` types to Trino `ipaddress` type.
+  ({issue}`7098`)
+* Allow mapping ClickHouse `fixedstring` or `string` as Trino `varchar` via the
+  `map_string_as_varchar` session property. ({issue}`10601`)
+* Disable `avg` pushdown on `decimal` types to avoid incorrect results.
+  ({issue}`10650`)
+* Fix spurious errors when metadata cache is enabled. ({issue}`10544`,
+  {issue}`10512`)
+
+## Druid connector
+
+* Fix spurious errors when metadata cache is enabled. ({issue}`10544`,
+  {issue}`10512`)
+
+## Hive connector
+
+* Add support for writing Bloom filters in ORC files. ({issue}`3939`)
+* Allow flushing cache for specific partition with the
+  [flush_metadata_cache](hive_flush_metadata_cache) system procedure.
+  ({issue}`10385`)
+* Add support for long lived AWS Security Token Service (STS) credentials for
+  authentication with Glue catalog. ({issue}`10735`)
+* Ensure transaction locks in the Hive Metastore are released in case of query
+  failure when querying Hive ACID tables.  ({issue}`10401`)
+* Disallow reading from Iceberg tables when redirects from Hive to Iceberg are
+  not enabled. ({issue}`8693`, {issue}`10441`)
+* Improve performance when reading ORC files with Bloom filters with range
+  predicates. ({issue}`4108`)
+* Fix spurious errors when metadata cache is enabled. ({issue}`10646`,
+  {issue}`10512`)
+* Prevent data loss during `DROP SCHEMA` when the schema location contains files
+  deep in the directory tree, but not tables. ({issue}`10485`)
+* Fix inserting into transactional table when `task_writer_count` > 1.
+  ({issue}`9149`)
+* Fix possible data corruption when writing data to S3 with streaming enabled.
+  ({issue}`10710 `)
+
+## Iceberg connector
+
+* Add Iceberg `$properties` system table.  ({issue}`10480`)
+* Add support for `ALTER TABLE .. EXECUTE OPTIMIZE` statement. ({issue}`10497`)
+* Respect Iceberg column metrics mode when writing. ({issue}`9938`)
+* Add support for long lived AWS Security Token Service (STS) credentials for
+  authentication with Glue catalog. ({issue}`10735`)
+* Improve performance when reading ORC files with Bloom filters with range
+  predicates. ({issue}`4108`)
+* Improve select query planning performance after write operations from Trino.
+  ({issue}`9340`)
+* Ensure table statistics are accumulated in a deterministic way from Iceberg
+  column metrics. ({issue}`9716`)
+* Prevent data loss during `DROP SCHEMA` when the schema location contains files
+  deep in the directory tree, but not tables. ({issue}`10485`)
+* Fix materialized view refresh when view a query references the same table
+  multiple times. ({issue}`10570`)
+ * Fix possible data corruption when writing data to S3 with streaming enabled.
```

Comment:
Maybe mention the relevant config name?

Reply:
Same in Hive

Reply:
it would be with hive.s3.streaming.enabled .. but this is documented and therefore should not be in the release notes. We just fixed an issue around it and not implemented it new or so.